regtests/t_pyspark/src/test_spark_sql_s3_with_privileges.py

```diff
@@ -508,6 +508,201 @@ def test_spark_credentials_can_delete_after_purge(root_client, snowflake_catalog
       pytest.fail(f"Expected all data to be deleted, but found data files {objects['Contents']}")
 
 
+@pytest.mark.skipif(os.environ.get('AWS_TEST_ENABLED', 'False').lower() != 'true', reason='AWS_TEST_ENABLED is not set or is false')
+def test_spark_credentials_can_write_with_random_prefix(root_client, snowflake_catalog, polaris_catalog_url, snowman,
```

Comment:
Is snowman included in Polaris?

Reply:
It's one of the test fixtures created earlier in the test - https://github.com/apache/polaris/blob/e1b07d958d9f2bb6c41bd37073016d39d1fef276/regtests/t_pyspark/src/test_spark_sql_s3_with_privileges.py#L42-L84